### PR TITLE
Remove C++ lowering path.

### PR DESCRIPTION
**Pull request by whchung**
This is PR is a draft. Remove C++ lowering path to reduce number of codes / tests to maintain.

### CODE REVIEW

**Comment by whchung** [comment]
@jerryyin would you mind help review this PR?

**Review by jerryyin — APPROVED** [review]

**Comment by jerryyin** [comment]
@whchung LGTM, once you merge this PR I will step up and work on the second part of it: On library + MIOpen

**Comment by whchung** [comment]
@jerryyin this PR has been merged. We have eliminated the C++ path.

**Pull request merged**
